src/jobs/build-app.yml

```diff
@@ -0,0 +1,30 @@
+description: >
+  This job will build an app
+
+executor: node/default
+
+parameters:
+  # Required Parameters
+  current-directory:
+    type: string
+    description: |
+      The current working directory
+
+steps:
+  - checkout
+  - node/install
+  - attach_workspace:
```

Comment:
why are we attaching? Was something persisted in a different job from the same workspace?